js/apps/admin-ui/src/user/UserForm.tsx

```diff
@@ -52,6 +52,7 @@ export type UserFormProps = {
   bruteForce?: BruteForced;
   userProfileMetadata?: UserProfileMetadata;
   save: (user: UserFormFields) => void;
+  refresh?: any;
```

Comment:
Best to give this a proper type
```suggestion
  refresh?: () => void;
```

Reply:
I had to add a check to make sure the `refresh` prop is defined before calling it. Hope it is OK.

Reply:
No problem!